FrameworkSystem/Client/SystemAdministratorClientCLI.py

```diff
@@ -419,7 +419,8 @@ def getInstallations( self, argss ):
           key = 'UnInstallationTime.smaller'
         elif arg == '-uta':
           key = 'UnInstallationTime.bigger'
-      else:
+      elif isinstance(key, str):
```

Comment:
I still think that it is ugly to put such "useless dust" in the code to make pylint happy